core/timetrace.go

```diff
@@ -310,3 +320,60 @@ func collides(toCheck Record, allRecords []*Record) bool {
 
 	return false
 }
+
+// ListIntegrations returns a set of all integrations available to timetrace
+func (t *Timetrace) ListIntegrations() map[string]Provider {
+	if len(t.integrations) < 1 {
+		return nil
+	}
+
+	return t.integrations
+}
+
+// VerifyPush confirms with the user the records that are about to be uploaded
+// to the selected integ
+func (t *Timetrace) VerifyPush(integrationName string) ([]*Record, error) {
+	// get all records locally TODO: obviously improve the time parsing
```

Comment:
This can be improved to have the user pass the "today" or "yesterday" etc as a flag. I hardcoded to today just for convenience.